libs/community/tests/integration_tests/document_loaders/parsers/test_images.py

```diff
@@ -58,3 +64,35 @@ def test_image_parser_with_differents_files(
     documents = list(blob_loader(**kw).lazy_parse(blob))
     assert len(documents) == 1
     assert re.compile(body).match(documents[0].page_content)
+
+
+@pytest.mark.parametrize(
+    "blob_loader,kw",
+    [
+        (RapidOCRBlobParser, {}),
+        (TesseractBlobParser, {}),
+        (
+            LLMImageBlobParser,
+            {
+                "model": FakeMessagesListChatModel(
+                    responses=[
+                        ChatMessage(
+                            id="ai1",
+                            role="system",
+                            content="A building. MAKE TEXT STAND  OUT FROM BACKGROUNDS",
+                        ),
+                    ]
+                )
+            },
+        ),
+    ],
+)
+def test_image_parser_with_numpy(
+    blob_loader: Type,
+    kw: dict[str, Any],
+) -> None:
+    blob = Blob.from_path(
+        path_base / "examples/gray.npy", mime_type="application/x-npy"
```

Comment:
Could we generate this in memory instead of using a file? We want to avoid relying on file fixtures as much as possible since the content is not easy to inspect or modify.